sympy/polys/rings.py

```diff
@@ -2531,15 +2531,140 @@ def compose(f, x, a=None):
 
         return poly
 
+
+    def coeff_wrt(self, x, dg):
+        """
+        Coefficient of ``f`` with respect to ``x**deg``.
+
+        Treating ``f`` as a univariate polynomial in ``x`` this finds the
+        coefficient of ``x**deg`` as a polynomial in the other generators.
+
+        Parameters
+        ==========
+
+        p : sympy.Poly
+            The polynomial to compute the expression for.
```

Comment:
There is no parameter `p`. It is `self` but we don't need to list `self` in the parameters of a method.